lib/manageiq/environment.rb

```diff
@@ -0,0 +1,108 @@
+require 'fileutils'
+require 'pathname'
+
+module ManageIQ
+  module Environment
+    APP_ROOT = Pathname.new(File.expand_path('../../..', __FILE__))
+
+    def self.ensure_config_files
+      config_files = {
+        "certs/v2_key.dev"        => "certs/v2_key",
+        "config/cable.yml.sample" => "config/cable.yml",
```

Comment:
Is there any way we can rename cable.yml.sample to cable.sample.yml?  The latter allows editors to syntax highlight while the former does not.  NOT necessarily for this PR.  cc @skateman